scipy/stats/_continuous_distns.py

```diff
@@ -7700,15 +7716,17 @@ def _entropy(self):
         for the root finder; the `scale` parameter and any other parameters
         for the optimizer are ignored.\n\n""")
     def fit(self, data, *args, **kwds):
+        if kwds.pop('superfit', False):
+            return super().fit(data, *args, **kwds)
         data, floc, fscale = _check_fit_input_parameters(self, data,
                                                          args, kwds)
 
-        def scale_mle(loc, data):
+        def scale_mle(loc):
```

Comment:
These functions are only defined within `fit`, and we never provide alternative values of `data`, so there's no need to accept it as a parameter.